playbooks/hc-ansible-deployment/tasks/gluster_deployment.yml

```diff
@@ -7,6 +7,44 @@
   gather_facts: no
   any_errors_fatal: true
 
+  # We are moving some of the prerequisite checks from gluster.features role so
+  # that we fail early even before backend setup
+
+  pre_tasks:
+    # Check if the hosts have valid hostnames
+    - name: Check if valid hostnames are provided
+      command:
+      args:
+        argv:
+          - getent
+          - ahosts
+          - "{{ item }}"
+      register: result
+      with_items: "{{ groups['all'] }}"
+      failed_when: result.stdout_lines|length == 0 and result.rc != 0
+      when: gluster_features_fqdn_check | default(true)
+      delegate_to: localhost
+      run_once: true
+
+    # Check if /var has enough disk. We check for 15G minimum.
+    - name: Check if /var/log has enough disk space
+      shell: df -m /var/log | awk '/[0-9]%/ {print $4}'
+      register: disk_size
+      failed_when: disk_size.stdout|int < gluster_features_min_disk|default(15360)
```

Comment:
are you changing the GiB to GB check here?

Reply:
Yes. I have done that.